src/common/CalculatorEngineCommon/ExprtkEvaluator.cpp

```diff
@@ -0,0 +1,46 @@
+#include "ExprtkEvaluator.h"
+#include "exprtk.hpp"
+#include <iomanip>
+#include <iostream>
+#include <sstream>
+
+namespace ExprtkCalculator::internal
+{
+
+    std::wstring ToWStringFullPrecision(double value)
+    {
+        std::wostringstream oss;
+        oss << std::fixed << std::setprecision(15) << value;
+        return oss.str();
+    }
+
+    std::string WStringToAscii(const std::wstring& wstr)
```

Comment:
Q: What is the purpose of this function? If it's going to convert a UTF-16 string into UTF-8, `winrt::to_string()` should be considered.